styles.css

```diff
@@ -1,4 +1,339 @@
-h1 {
-  color: #627cf5;
-  font-size: 100px;
+body {
+  /* Template 1-Mobile-Main */
+
+  position: relative;
+  width: 375px;
+  height: 5410px;
+
+  /* Neutrals/N10 */
+
+  background: #f7f7f9;
+}
+
+.status-bars {
+  /* Bars/Status Bars/ Light */
+
+  position: absolute;
+  width: 375px;
+  height: 44px;
+  left: calc(50% - 375px / 2);
+  top: 0;
+}
+
+.app-bar {
+  /* App Bar - Compact - Dark - iOS */
+
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.toolbar {
+  /* Toolbar - Dark - Compact */
+
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+.normal-button {
+  /* Normal Button/Tertiary/Enabled */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.burger {
+  /* Icons/Icon - Menu */
+
+  position: static;
+  left: 20%;
+  right: 20%;
+  top: 20%;
+  bottom: 20%;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.logo {
+  /* Logo Placeholder */
+
+  /* Auto layout */
+
+  display: block;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+}
+
+.logo > span {
+  /* My Logo */
+
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.toolbar-container {
+  /* Toolbar Container */
+
+  position: absolute;
+  visibility: hidden;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  /* Base / 900 */
+
+  background: #181e25;
+}
+
+.headline {
+  /* Headline */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  background: url("assets/bg.png") repeat-y;
```

Comment:
For the background image please make sure you stick to the Figma design and it's should cover the whole width of the screen as in the design. Kindly, fix that here I attached your design along with the Figma design to show how the background image.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="382" alt="image" src="https://user-images.githubusercontent.com/34578631/170270140-83750f12-cbee-4a83-8657-d94ff823a09b.png"> | <img width="382" alt="image" src="https://user-images.githubusercontent.com/34578631/170270072-4322bbdf-a6c1-4583-91bc-d3914e14e10a.png"> |